src/config/complex_templates.m4

```diff
@@ -38,6 +38,8 @@ template<typename T> class X {
 // Test if complex multiply on this type works
 std::complex< X<double> > a, b, c;
 c = a * b;
+// Test if complex norm works
+c = std::norm(c);
```

Comment:
Actually this line should probably more properly be `X<double> d = std::norm(c)`.